pkg/backend/backend.go

```diff
@@ -57,6 +63,21 @@ func NewBackend(ctx context.Context, log *logrus.Entry, env env.Interface, db *d
 	b.ocb = &openShiftClusterBackend{backend: b}
 	b.sb = &subscriptionBackend{backend: b}
 
+	if len(os.Getenv("MAX_WORKERS")) > 0 {
```

Comment:
*if* we do this, I'd like to get the Getenv's into pkg/env, so they are all in one place.

Reply:
How would you imagine this? I dont want to add more to this interface as you expressed interest not to add anything to this anymore
```
	CosmosDB() (string, string)
	DatabaseName() string
	DialContext(context.Context, string, string) (net.Conn, error)
	Domain() string
	FPAuthorizer(string, string) (autorest.Authorizer, error)
	GetSecret(context.Context, string) (*rsa.PrivateKey, []*x509.Certificate, error)
	Listen() (net.Listener, error)
	Zones(vmSize string) ([]string, error)
```

I would imagine something like 
```
type BackedConfig{
...
}
```
and put it inside the `env` interface as `BackendConfig`

Reply:
At some point, we will need some configuration capability for individual components. Avoiding it now we just pushing this for the future. We can always refactor...